helpful_tools/Install-BuildKit-GitHubActions/workflow.yaml

```diff
@@ -0,0 +1,33 @@
+name: Build

```

Comment:
nit: perhaps just calling the directory `GithubActions`? and then this file `install_buildkit_workflow.yaml`?
This is just to avoid the confusion that this could be a powershell module, since the rest with this kind of casing/wording are powershell modules...

Reply:
Makes sense, fixed